Script/index.js

```diff
@@ -54,4 +54,35 @@ document.addEventListener("keydown", function (e) {
   }
 });
 
-// Mobile version of Overlay
+// Check email is valid
+const form = document.getElementById("formD");
+const formB = document.getElementById("FMB");
+const email = document.querySelector(".email");
+
+function checkEmail(input) {
+  const re =
+    /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  if (re.test(input.value.trim())) {
+    email.style.borderColor = "#2ecc71";
+  } else {
+    email.style.borderColor = "#e74c3c";
+  }
+}
+
+// Check required fields
+function checkRequired(inputArr) {
+  inputArr.forEach(function (input) {
+    if (input.value.trim() === "") {
+      showError(input, `${getFieldName(input)} is required`);
+    } else {
+      showSuccess(input);
+    }
+  });
+}
+
+// Event listeners
+form.addEventListener("submit", function (e) {
+  e.preventDefault();
+
+  checkEmail(email);
+});
```

Comment:
- [ ] You did an excellent job on this project, but you still need to improve on something in order for it to be approved. As required for this project, the email should be all lowercase, and if not, it should display some validation message indicating that the email should be in lowercase. However, your form is still accepting uppercase for email, so please fix this for your project to reach the minimum for this milestone. :wink:  :wink: 
check this please :point_down: 
![upcase](https://user-images.githubusercontent.com/88894525/195987241-9fe16c9d-0cab-47a0-a62d-7bddc614bb62.png)